src/bindPolyfill.js

```diff
@@ -1,29 +0,0 @@
-(function() {
```

Comment:
i would love some feedback on this. i don't think we should be including polyfills in a library. my view is that should be the responsibility of the application (it knows what browsers are supported and can decide whether to polyfill).

should we continue to include this in the UMD artifact for backwards compatibility?

Reply:
@samandmoore what's your take on this question?

Reply:
I'm fine with this choice. It's tricky because we'd wanna document that you need this polyfill in some cases otherwise folks won't know until their app blows up. So, maybe put this in the readme along with jquery, jquery cookie, and whatever else the consumer needs to BYO